components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -580,6 +614,266 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
         return;
     }
 
+    /**
+     * This method is used to append the application side requested scopes after validating.
+     * The application can request the optional scopes either via adaptive scripts or via the authorize
+     * request query parameters.
```

Comment:
Shall we mention that the adaptive script will get priority?

Reply:
Fixed 55d730c53c65381c2320598caa84bb9f7256b08e